# HG changeset patch
# User Rik <rik@octave.org>
# Date 1419977093 28800
#      Tue Dec 30 14:04:53 2014 -0800
# Node ID dbdc60e36dbc7e31ca2dbc62d342b451d57854d3
# Parent  0b4915fcd2eb8d5eb47bdc3cd7ae95ab6d06feca
Don't distribute mxarray.h in tarball (bug #43805).

* libinterp/corefcn/module.mk: Remove mxarray.h from COREFCN_INC include files.

diff --git a/libinterp/corefcn/module.mk b/libinterp/corefcn/module.mk
--- a/libinterp/corefcn/module.mk
+++ b/libinterp/corefcn/module.mk
@@ -71,17 +71,16 @@ COREFCN_INC = \
   corefcn/ls-mat-ascii.h \
   corefcn/ls-mat4.h \
   corefcn/ls-mat5.h \
   corefcn/ls-oct-ascii.h \
   corefcn/ls-oct-binary.h \
   corefcn/ls-utils.h \
   corefcn/mex.h \
   corefcn/mexproto.h \
-  corefcn/mxarray.h \
   corefcn/oct-errno.h \
   corefcn/oct-fstrm.h \
   corefcn/oct-handle.h \
   corefcn/oct-hdf5.h \
   corefcn/oct-hist.h \
   corefcn/oct-iostrm.h \
   corefcn/oct-lvalue.h \
   corefcn/oct-map.h \
